src/main.js

```diff
@@ -9,6 +9,7 @@ import store from './store'
 import BootstrapVue from 'bootstrap-vue'
 import { apolloProvider } from './graphql/client'
 import globalAlert from '@/mixins/globalAlert'
+import './styles/index.css'
```

Comment:
Looks like this is a huge file - does this contain the tailwind framework? Is it correct to store it or should it get generated when we build the package?

Reply:
Yes, you are right. The development build is usually large by design. According to tailwind documentation, to make the development environment more flexible, tailwind generates thousands of utility classes which most of which we probably don't use. But while buiding in production enviornment, Tailwind optimizes final build size by removing unused styles.